jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml

```diff
@@ -2,28 +2,41 @@
   <!-- mainly copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
   <arg name="enable_auto_dock" default="true" />
   <arg name="joy_device" default="/dev/ps3joy"/>
-  <arg name="disable_arm_teleop" default="true"/>
 
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
   <node name="joy" pkg="joy" type="joy_node" respawn="true">
     <param name="autorepeat_rate" value="1"/>
     <param name="dev" value="$(arg joy_device)"/>
   </node>
 
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
+  <!-- copied to support unsafe teleop. -->
```

Comment:
need to launch `joystick_teleop` because of launch_teleop:=false is set at /etc/ros/indigo/robot.launch, see fetchrobotics/fetch_robots#42

Reply:
- copied to support unsafe teleop
+ `joystick_teleop` need to publish `/teleop/cmd_vel/unsafe` , see https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/jsk_fetch_startup/README.md

Reply:
I added the comments.